specification/gedcom.md

```diff
@@ -176,8 +176,8 @@ Use of Unicode-aware processing and display libraries is recommended.
 
 Character-level grammars are specified in this document using
 Augmented Bakaus-Naur Form (ABNF)
-as defined in [STD 68](https://www.rfc-editor.org/info/std68)
-and modified in [RFC 7405](https://www.rfc-editor.org/info/rfc7405).
+as defined in STD 68 (<https://www.rfc-editor.org/info/std68>)
+and modified in RFC 7405 (<https://www.rfc-editor.org/info/rfc7405>).
```

Comment:
This regresses PR #9.  The original is more correct so recommend reverting this change.